pkg/util/log/slog/formatters/file.go

```diff
@@ -0,0 +1,51 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package formatters
+
+import (
+	"os"
+	"path/filepath"
+	"runtime"
+	"strings"
+)
+
+// ShortFilePath returns the short path of the file that the log message was emitted from.
+func ShortFilePath(frame runtime.Frame) string {
+	return extractShortPathFromFullPath(frame.File)
+}
+
+func extractShortPathFromFullPath(fullPath string) string {
+	shortPath := ""
+	if strings.Contains(fullPath, "-agent/") {
+		// We want to trim the part containing the path of the project
+		// ie DataDog/datadog-agent/ or DataDog/datadog-process-agent/
+		slices := strings.Split(fullPath, "-agent/")
+		shortPath = slices[len(slices)-1]
```

Comment:
No need to read the string twice
```suggestion
	slices := strings.Split(fullPath, "-agent/")
	if len(slices) > 1 {
		// We want to trim the part containing the path of the project
		// ie DataDog/datadog-agent/ or DataDog/datadog-process-agent/
		shortPath = slices[len(slices)-1]
```